dev_guide/managing_images.adoc

```diff
@@ -586,6 +586,235 @@ instead of listing all image streams.
+token, to access content on the new registry.
+
+All image streams are updated to point to the new registry. Because the
+new registry requires authentication for access, there is a new secret in the
+{product-title} namespace called `imagestreamsecret`. When you run your build,
+your `imagestreamsecret` gets populated with your `kubernetes.io/dockerconfigjson`
+credentials.
+
+There are two places your credentials must exist:
+
+* *{product-title} namespace*. Your credentials must exist in the {product-title}
+namespace so that the image streams in the {product-title} namespace can import.
+* *Your host*. Your credentials must exist on your host because Kubernetes will
+use the credentials from your host when it goes to pull images.
+
+As long as you have the following items in place, you should be able to
+access the new registry.
+
+* You should see image import secret, `imagestreamsecret`,  in your {product-title}
+namespace. That secret should have credentials that will allow you to access
+the new registry.
+* All of your cluster nodes should have a `dockerconfigjson` that allow you to
+access the Red Hat registry.
+
+[NOTE]
+====
+RHEL 8 will only be available on the new registry.
+====
+
+[[create-user-accounts]]
+=== Creating User accounts
+
+//Concept
+
+If you are a Red Hat customer with entitlements to Red Hat products, you have
+an account with applicable user credentials. These are the username and password
+that you use to log in to the Red Hat Customer Portal.
+
+If you do not have an account, you can acquire one for free by registering for
+one of the following options:
+
+* link:https://developers.redhat.com/[*Red Hat Developer Program*]. This account
+gives you access to developer tools and programs.
+* link:https://access.redhat.com/products/red-hat-enterprise-linux/evaluation[*30-day Trial Subscription*].
+This account gives you a 30-day trial subscription with access to select Red Hat
+software products.
+
+
+[[create-authentication-tokens]]
+=== Creating Authentication Tokens
+
+You must create token if your organization manages shared accounts.
+Administrators can create, view, and delete all tokens associated an
+organization.
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+
+//[[Procedure]]
+*Procedure*
+
+To create a token in order complete a `docker login`:
+
+. Navigate to `registry.redhat.io`.
+. Log in with your RHN username and password.
+. Accept terms when prompted.
+.. If you are not immediately prompted to accept terms, you will be prompted
+when proceeding with the following steps.
+. From the *Registry Service Accounts* page, click *Create Service Account*
+.. Provide a name for the service account. It will be prepended with a random string.
+.. Enter a description.
+.. Click create.
+. Navigate back to your Service Accounts.
+. Click the Service Account you created.
+. Copy the username, including the prepended string.
+. Copy the token.
```

Comment:
this section needs to somehow tie into the installation instructions.  Basically you'll need to go get this username+token and then paste it into the appropriate ansible vars in your inventory file.

Again @sdodson and @michaelgugino can provide specifics.